terraform/anomaly_detection/main.tf

```diff
@@ -0,0 +1,213 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+locals {
+  dataflow_service_account = "my-dataflow-sa"
+  max_dataflow_workers     = 1
+  worker_disk_size_gb      = 200
+  machine_type             = "g2-standard-4"
```

Comment:
Will the anomaly detection pipeline use a GPU? If it will, then we can leave this machine type as default. If not, then I would remove this and would use a specific machine type. This machine type is used for GPU workloads.

Reply:
I see the suggestion to use gpu's for anomaly detection in solution guide

Reply:
Ah, good then, I was not aware we recommend that in the guide.